Wikipedia/Code/TempAccountSheetCoordinator.swift

```diff
@@ -81,7 +83,21 @@ final class TempAccountSheetCoordinator: Coordinator {
                 },
                 didTapDone: didTapDone,
                 ctaTopButtonAction: {
-                    
+                    let loginCoordinator = LoginCoordinator(navigationController: self.navigationController, theme: self.theme)
```

Comment:
Please revert these login changes so that this PR is only about the toast presentation.